Essentials/src/com/earth2me/essentials/commands/Commanddelkit.java

```diff
@@ -35,7 +31,15 @@ public void run(final Server server, final CommandSource sender, final String co
             }
 
             ess.getKits().removeKit(kitName);
-            sender.sendMessage(tl("deleteKit", kit));
+            sender.sendMessage(tl("deleteKit", kitName));
         }
     }
+
+    @Override
+    protected List<String> getTabCompleteOptions(final Server server, final CommandSource sender, final String commandLabel, final String[] args) {
+        if (args.length == 1) {
+            return Lists.newArrayList(ess.getKits().getKits().getKeys(false));
```

Comment:
The fact that this is how we get the name of all kits makes me grumpy.

Reply:
I have yet to figure out how the kit code works.